libs/zephyr-rspack-plugin/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@module-federation/automatic-vendor-federation": "1.2.1",
-    "@rspack/core": "^1.1.3",
+    "@rspack/core": "^1.2.2",
```

Comment:
why this version?

Reply:
There's a type mismatch between the versions we use of Rspack on everything else and the version ModernJS expects as a type.

Without the override, I cannot build the bundle properly. Maybe @ryok90 knows of a better way of doing it, a pnpm override is no bueno

Reply:
nvm, this version is indeed unnecessary. Still need to address the pnpm overrides though